bifrost/app/blog/blogs/debugging-chatbots-and-ai-agents-with-sessions/src.mdx

```diff
@@ -0,0 +1,171 @@
+Have you ever wondered at which stage in the multi-step process does your AI model start hallucinating? Perhaps you’ve noticed consistent issues with a specific part of your AI agent workflow?
+
+![Debugging RAG Chatbots and AI Agents with Sessions](/static/blog/sessions-ui.webp)
+
+These are common questions developers face when building **AI agents** and **Retrieval Augmented Generation (RAG)** chatbots. Getting reliable responses and minimizing errors like hallucination is incredibly challenging without visibility into how users interact with your Large Language Model (LLM). 
+
+Addressing these pitfalls is crucial in creating robust and reliable AI agents and RAG chatbots. In this blog, we will delve into how to use **<span style={{color: '#0ea5e9'}}>Helicone's Sessions</span>** feature to help you maintain context, reduce errors, and improve the overall performance of your LLM app. 
+
+### What you will learn: 
+
+- AI agents and how they work
+- Challenges of debugging AI agents
+- Introduction to Sessions
+- Using Sessions in Helicone
+- How industries use Sessions to debug AI agents
+
+Please skip ahead to **Using Sessions in Helicone** if you are familiar with AI agents.  
+
+---
+
+## What are AI agents? 
+
+An AI agent is a software program that **<span style={{color: '#0ea5e9'}}>autonomously performs specific tasks using advanced decision-making abilities</span>**. It interacts with its environment by collecting data, processing it, and deciding on the best actions to achieve predefined goals. 
```

Comment:
This is more focused to AI agents in general. We should be talking about AI agents powered by LLMs

Reply:
https://learn.microsoft.com/en-us/azure/cosmos-db/ai-agents